cranelift/filetests/src/zkasm_codegen.rs

```diff
@@ -35,13 +35,16 @@ pub fn generate_zkasm(settings: &ZkasmSettings, wasm_module: &[u8]) -> String {
 
     let mut program: Vec<String> = Vec::new();
 
-    let start_func = zkasm_environ
+    let main_func = zkasm_environ
         .info
-        .start_func
-        .expect("Must have a start function");
+        .functions
+        .iter()
+        .find(|&(_, exportable_func)| exportable_func.export_names.contains(&"main".to_string()))
+        .expect("Must have a `main` function");
```

Comment:
Should we also check the signature of the function?

Reply:
That's a good idea, added an assert.